.github/renovate.json

```diff
@@ -8,7 +8,8 @@
     {
       "fileMatch": [
         ".*\\.Dockerfile$",
-        ".*doc/packaging\\.md$"
+        ".*doc/packaging\\.md$",
+        ".*generator/internal/scaffold_generator\\.cc$"
```

Comment:
This is not quite right.  We use a different manager to update the `.bzl` files:

https://docs.renovatebot.com/modules/manager/bazel/

It needs to do fancy stuff with the SHA256 too.

Reply:
renovate-bot doesn't believe me when I say `scaffold_generator.cc` is a WORKSPACE file. I give up. Closing.